content/docs/reference-events.md

```diff
@@ -60,62 +58,62 @@ function onClick(event) {
 }
 ```
 
-> Note:
+> Not:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
-
-## Supported Events {#supported-events}
-
-React normalizes events so that they have consistent properties across different browsers.
-
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
-
-- [Clipboard Events](#clipboard-events)
-- [Composition Events](#composition-events)
-- [Keyboard Events](#keyboard-events)
-- [Focus Events](#focus-events)
-- [Form Events](#form-events)
-- [Mouse Events](#mouse-events)
-- [Pointer Events](#pointer-events)
-- [Selection Events](#selection-events)
-- [Touch Events](#touch-events)
-- [UI Events](#ui-events)
-- [Wheel Events](#wheel-events)
-- [Media Events](#media-events)
-- [Image Events](#image-events)
-- [Animation Events](#animation-events)
-- [Transition Events](#transition-events)
-- [Other Events](#other-events)
+> Olay özelliklerine asenkron bir şekilde erişmek isterseniz, olay üzerinde `event.persist()` çağırmalısınız. Bu şekilde sentetik olay havuzdan çıkarılır ve olay referanslarının kullanıcı kodu tarafından korunmasına olanak sağlanır.
+
+## Desteklenen Olaylar {#supported-events}
+
+React olayları normalleştirir ve olaylar bu şekilde farklı tarayıcılarda tutarlı özelliklere sahip olur.
+
+Aşağıdaki olay yöneticileri, balonlanma(bubbling) evresinde bir olay tarafından tetiklenir. Yakalama evresine bir olay yöneticisi kaydetmek için, olay ismine `Capture` ilave edin. Örneğin tıklama olayını, yakalama evresinde yönetmek için `onClick` kullanmak yerine `onClickCapture` kullanın.
+
+- [Not Panosu Olayları](#clipboard-events)
+- [Dizim Olayları](#composition-events)
```

Comment:
In the context of this event type, the words `bilesim`, `birlesim` or even `duzenleme` might be more accurate. 

From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CompositionEvent) -> `The DOM CompositionEvent represents events that occur due to the user indirectly entering text.`

From [StackOverflow](https://stackoverflow.com/a/51413620/1510671) ->  `Composition Events might be used to add accents to characters despite their absence from standard US keyboards, to build up logograms of many Asian languages from their base components or categories, to select word choices from a combination of key presses on a mobile device keyboard.`

What do you think ?

Reply:
I think `Kompozisyon` can be used for `Composition`. You can check this word on [tdk](http://www.tdk.gov.tr):
![image](https://user-images.githubusercontent.com/1326963/53787591-7414a180-3f30-11e9-9128-e91de4ac3317.png)

Reply:
If TDK says so, I can use `Kompozisyon` for this to keep it simple. Thanks for your valuable comments @alioguzhan